keyboards/winkeyless/bface/keymaps/p3lim/keymap.c

```diff
@@ -0,0 +1,156 @@
+/*
+Copyright 2019 Adrian L Lange <legal@p3lim.net>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include QMK_KEYBOARD_H
+#include "quantum.h"
+
+#define LAYOUT_p3lim(\
```

Comment:
This looks to be an iso layout, can you contribute this to the board instead of just your keymap? You can just copy and paste it into bface.h and name it something more generic.

Reply:
I looked at this when I was reviewing earlier, but according to @p3lim's readme and keymap comments, it's not actually fully ISO.

The layout actually seems to be 60% ISO + ANSI Left Shift + Split Right Shift + Tsangan bottom row.

Reply:
Correct, I'm a bit weird.

Reply:
As such, I think this should be best left here, then.

Also, 
>Correct, I'm a bit weird.

One of us, one of us!  :D